api/pom.xml

```diff
@@ -21,11 +21,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.glassfish</groupId>
-        <artifactId>json</artifactId>
-        <version>2.0.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.6</version>
     </parent>
+    
+    <prerequisites>
```

Comment:
this results in new warning:
```
[WARNING] The project org.glassfish:json:pom:2.0.0-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
```

Reply:
I have created another PR with the fix:
https://github.com/eclipse-ee4j/jsonp/pull/229